recipes/lora_finetune_distributed.py

```diff
@@ -351,10 +362,17 @@ def setup(self, cfg: DictConfig) -> None:
         self._profiler = self._setup_profiler(cfg.get(PROFILER_KEY, None))
 
         # Used to ignore labels for loss computation
+        bzs_cache = (
```

Comment:
```suggestion
        bsz_cache = (
```

Reply:
Thanks, addressed in [4436592](https://github.com/pytorch/torchtune/pull/2464/commits/4436592dd01a9fa329b0ce65b0e529e869240352)